service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -646,6 +647,30 @@ public String updateGcpControlledResourcesRegionAsync(
         .addParameter(IS_WET_RUN, wetRun)
         .submit();
   }
+  // TODO (PF-2269): Clean this up once the back-fill is done in all Terra environments.
+  @Traced
+  @Nullable
+  public String updateGcpControlledBigQueryDatasetsLifetimeAsync() {
```

Comment:
Why does this return anything? No one is waiting.
If there is a reason, make a comment.

Reply:
+1. my update region methods are fed into api endpoints. but if we are doing this at startup, we don't need to return jobid.

Reply:
it is easier for testing though..

as this is async, the test will need a way to pull the job till it's done.